source/lib/neopixelsend.s

```diff
@@ -27,7 +27,7 @@
 
 .global sendNeopixelBuffer
 
- /* declared as extern void sendBuffer(uint32_t pin, uint8_t* data_address, uint16_t num_leds) */
+ /* declared as extern void sendBuffer(uint32_t pin, uint8_t* data_address, uint16_t num_leds, uint16_t num_channels) */
```

Comment:
can just change the `num_leds` parameter to be `num_bytes`, then the multiplication of channels*LEDs can be done outside this routine